CMakeLists.txt

```diff
@@ -141,6 +147,15 @@ endif ()
 
 install (TARGETS iwasm_static ARCHIVE DESTINATION lib)
 
+# If it's a Release build, strip the static library
+if (CMAKE_BUILD_TYPE STREQUAL "Release")
```

Comment:
Had better also check CMAKE_STRIP, e.g. `if (CMAKE_STRIP AND CMAKE_BUILD_TYPE STREQUAL "Release")`?

Reply:
Yes, I added `CMAKE_STRIP` check both conditions that I also check Release type.